intuita-webview/src/CreateIssue/index.tsx

```diff
@@ -14,6 +14,17 @@ import './tiptap.css';
 
 const lowlight = createLowlight(common);
 
+const convertHTMLCodeBlockToMarkdownString = (htmlSnippet: string) => {
+	return htmlSnippet.replace(
```

Comment:
you can remove the block and the return statement

Reply:
addressed